lib/miq_automation_engine/engine/miq_ae_domain_search.rb

```diff
@@ -35,7 +37,9 @@ def search(uri, ns, klass, instance, method)
         end
       end
       @partial_ns << ns unless @partial_ns.include?(ns)
-      find_first_fq_domain(uri, ns, klass, instance, method)
+      updated_ns = find_first_fq_domain(uri, "#{@prepend_namespace}/#{ns}", klass, instance, method) if @prepend_namespace
```

Comment:
Prefer something like a `File.join` method or equivalent that can handle tailing `/`.  This way, if someone passes in a tailing slash it won't create a bad URI.